### PR TITLE
minor: cleanups for ant-phase-verify

### PR DESCRIPTION
**Pull request by rnveach**
Minor found at https://github.com/sevntu-checkstyle/sevntu.checkstyle/pull/916

### CODE REVIEW

File: config/ant-phase-verify.xml [hunk 1]
```diff
@@ -12,7 +12,6 @@
              classpath="${mvn.runtime_classpath}"
              />
 
-    <property name="checkstyle.pattern.todo" value="NOTHingWillMatCH_-"/>
```
Comment (rnveach) [inline config/ant-phase-verify.xml:15]: Been around for 12 years at 86cbe5bd160285d46535c03e07a197fc542727ce but never been used.

File: config/ant-phase-verify.xml [hunk 2]
```diff
@@ -38,7 +37,6 @@
       <formatter type="plain"/>
       <formatter type="xml" toFile="${mvn.project.build.directory}/cs_errors.xml"/>
       <formatter type="sarif" toFile="${mvn.project.build.directory}/cs_errors.sarif"/>
-      <classpath path="${mvn.runtime_classpath}"/>
```
Comment (rnveach) [inline config/ant-phase-verify.xml:41]: Assume this is just left over when we had checks against the classpath.

File: pom.xml [hunk 3]
```diff
@@ -1496,21 +1496,12 @@
...
-                <property name="project.basedir" value="${project.basedir}" />
                 <ant antfile="config/ant-phase-verify.xml" />
               </target>
             </configuration>
           </execution>
         </executions>
-        <dependencies>
-          <dependency>
-            <groupId>org.apache.ant</groupId>
-            <artifactId>ant-nodeps</artifactId>
```
Comment (rnveach) [inline pom.xml:1510]: Ant with no dependencies. It was integrated with normal ant.

File: config/ant-phase-verify.xml (hunk 2 above)
Comment (romani) [reply config/ant-phase-verify.xml:41]: I thought it is required, as we load all check by classpath not a jar

Comment (rnveach) [reply config/ant-phase-verify.xml:41]: Please note this is a custom property sent to our Checkstyle Ant. It is not related to Java's classpath loader.
https://github.com/checkstyle/checkstyle/blob/527578f9a89abd7136552763bf942f9cb334c203/src/main/java/com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTask.java#L92

Java's classpath loader is at https://github.com/checkstyle/checkstyle/blob/af99c7d552ee8470cd07590211e6cdf1a8881050/config/ant-phase-verify.xml#L12 .

If you examine the code, you will see this property is only used to display as an error if an exception occurs.
https://github.com/checkstyle/checkstyle/blob/527578f9a89abd7136552763bf942f9cb334c203/src/main/java/com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTask.java#L430-L431

Technically this whole property can be marked as deprecated.

I haven't searched the history, but this seems like its just for check's to access the real classpath.

Comment (rnveach) [reply config/ant-phase-verify.xml:41]: https://github.com/checkstyle/checkstyle/blob/b8ca6a585b824e91b3b8c72dd5cc53c0eb0ab0f1/src/main/java/com/puppycrawl/tools/checkstyle/CheckStyleTask.java#L362-L365
Original usage of the classpath property that is now removed.

Comment (rnveach) [reply config/ant-phase-verify.xml:41]: Code was removed at 
https://github.com/checkstyle/checkstyle/commit/145160f5e21b80c27dc93a1904fe33b9afd4f212#diff-56ee8828e56742120dd2db0061ea82707b7b27c5c0f9e9963a8cefe3aa2dbc3aL405
as part of https://github.com/checkstyle/checkstyle/issues/3773

**Review by romani — APPROVED** [review]
Ok to merge

**Pull request merged**